assets/app/view/game/corporation.rb

```diff
@@ -638,9 +638,10 @@ def render_loans
           loan_props[:style][:color] = contrast_on(color)
         end
 
+        corp_loans_text = @corporation.respond_to?(:corp_loans_text) ? @corporation.corp_loans_text : 'Loans'
```

Comment:
Why is the text in the corporation? Shouldn't it be derived from the step?

Reply:
I just thought it made sense to put it in the game's corporation file since this was also a corporation file.
I can change it.

Reply:
It's possible I'm misunderstanding, but since this info was already in /assets/app/view/game/corporation.rb, doesn't it make more sense to do this in the Corporation?

Reply:
My issue is that the corporation class is being queried. It shouldn't be used. Get the text from game. Set the default in the base game class to `Loans`, so you don't have to use `respond_to?`

Reply:
done